content/root/platform-customization.textile

```diff
@@ -0,0 +1,147 @@
+---
+title: Platform Customization
+section: root
+index: 45
+meta_description: "How Enterprise customers can create a custom environment and benefit from Active Traffic Management and other advanced Ably features."
+meta_keywords: "Ably, clusters, isolation, custom, CNAME, active traffic management"
+jump_to:
+  Contents:
+    - Active traffic management
+    - Dedicated and isolated clusters
+    - Regional routing and storage
+    - Setting up a custom environment
+---
+
+If you're an "Ably Enterprise":https://ably.com/pricing customer, then you can optionally create a *custom environment* to tailor the Ably platform to your specific requirements. A custom environment enables you to benefit from the following:
+
+- *Active traffic management*: Let Ably actively monitor not only the global cluster health but also your own endpoints and proactively take steps to isolate or move traffic to ensure business continuity.
+- *Dedicated, isolated clusters*: Rely upon guaranteed capacity and isolation from noisy neighbors.
+- *Regional routing of traffic*: Require that all traffic is routed and serviced within specific regions.
+- *Regional message storage*: Require that all messages are stored within a specific region.
+
+A custom environment provides you with your own endpoints to service Ably platform requests. These can be either subdomains of @ably.io@, or point at your own custom CNAME domain. If you choose the latter option, we can also customize our client library SDKs to use your chosen domain and make them available via our CDN as @cdn.ably.io/lib/yourcompany.min.js@.
+
+If you are not currently an enterprise customer but are interested in finding out more, then please "get in touch":https://ably.com/contact.
+
+If you're ready to enable a custom environment to take advantage of these features, then please "see our guide":#custom-environment-setup. Otherwise, read on to find out more.
+
+
+h2(#active-traffic-management). Active traffic management
+
+Active traffic management (ATM) is a feature available exclusively to our Enterprise customers that enables the Ably engineering team to automatically and manually manage the endpoints that customers connect to.
+
+This ensures that under adverse conditions, such as a denial of service attack or a major outage, Ably can proactively route Enterprise customer traffic to specific data centers or regions that are unaffected, or even to new temporary clusters. This enables customers to maintain service while the issue is investigated and resolved.
+
+This service is ideal for customers who want the piece of mind that Ably actively monitors not only the global cluster health but also the endpoints used by that customer, and can act to isolate or move traffic, ensuring business continuity.
```

Comment:
"piece" of mind.

I'd be in favour of removing idioms though :-)

Reply:
Haha! Good spot :)